### ISSUE TITLE
SignUp / Domains: make domain availability usable with a screen reader

### ISSUE THREAD

**Issue by rodrigoi** [Automattic/wp-calypso#23649]
Follow up on #21658.

Testing with VoiceOver on macOS.

When a domain search is performed, there are warnings about the availability of a domain that are not announced by the VoiceOver utility.

For example.

| Warning | |
| - | - |
| Domain is taken | ![screen shot 2018-03-26 at 13 07 44](https://user-images.githubusercontent.com/233601/37918298-fd4f443a-30f6-11e8-9e5d-ab274dbf85a1.png) |
| TLD not offered | ![screen shot 2018-03-26 at 13 08 10](https://user-images.githubusercontent.com/233601/37918320-12ef8fde-30f7-11e8-8c32-fdb0c8e75fc4.png) |
| Trademark Warnings | ![screen shot 2018-03-26 at 13 07 11](https://user-images.githubusercontent.com/233601/37918351-2a8cc5da-30f7-11e8-8e10-2aa8dbd111b2.png) |

This could be fixed by changing the way the search results components render, so these areas can be trated as `aria-live` regions with the correct role.

**Comment by youbrokesomething** [comment]
I can no longer reproduce this issue. I believe it was fixed in 2018.
